### ISSUE TITLE
Test run fails on import

### ISSUE THREAD

**Issue by hcl734** [firsttris/vscode-jest-runner#101]
Using default settings for jest-runner on an nx-setup repo.
Seems to work fine, just when I try to import library files, I get 
Cannot find module '...'

Working fine when running the test file with nx test or ng test.
What do I have to configure to make this work?

**Comment by firsttris** [comment]
i don't have xp with nx/ng test but...  my understanding its just a wrapper for jest in the angular cli.

jest-runner is basically creating the jest commands like you would regularly execute them over your terminal.

if such a command fails i could have something todo with your test/jest config?

**Comment by hcl734** [comment]
I installed jest-cli now and run the test with:
jest my-file.component.spec.ts
This gives the same error of missing component.

I had to add  
"modulePaths": [
  "./"
],
in the root level jest.config.js

To let jest also search for the modules defined in my root.
Furthermore I am using alias so I had to add:
  moduleNameMapper: {
    '^@my-lib/(.*)$': '<rootDir>/libs/$1/src/lib/$1.module',
  },


But even then more dependencies from zone.js etc. are requested, so I think I am giving up at this point and going back to nx test, where it works fine.
Thanks for your help though.
